Cleaning up the Get Involved page

pamoroso opened the pull request:
This change reorganizes and clarifies the [Get Involved](https://interlisp.org/project/getinvolved) page of the project site to address [issue #2217](https://github.com/Interlisp/medley/issues/2217).

pamoroso committed:
Clean up page Get Involved

Revise the Get Involved page as per issue #2217 to clarify some link anchors; identify the GitHub repo of Interlisp.github.io; remove the Mastodon and Bluesky links from section Chat and move the project's email details to section EMail; rename section Chat to Online communities and add a call for volunteers to monitor those platforms; and shorten the Weekly meetings heading by moving part of it to the section text. Point out that we have official accounts on Mastodon and Bluseky and no presence on Facebook.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

stumbo commented on content/en/project/getInvolved/_index.md:
```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
 * [Medley](https://github.com/Interlisp/medley): Lisp code, build scripts and GitHub automation
 * [maiko](https://github.com/Interlisp/maiko): The C-coded virtual machine
-* [this web site](https://github.com/Interlisp/Interlisp.github.io#README)
+* [the Interlisp.github.io repository](https://github.com/Interlisp/Interlisp.github.io#README) of the interlisp.org web site
```
How about keeping it consistent with the Medley and Maiko sites (should Maiko be capitalized?)
[Interlistp.github.io](xxx): The Hugo source code for the interlisp.org site

stumbo commented on content/en/project/getInvolved/_index.md:
```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
 
 * [Medley](https://github.com/Interlisp/medley): Lisp code, build scripts and GitHub automation
 * [maiko](https://github.com/Interlisp/maiko): The C-coded virtual machine
```
The text above this mentions medley, maiko and online.  Should we add a link to the online repo here?

stumbo commented on content/en/project/getInvolved/_index.md:
```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
+### Social media accounts
 
-You can follow us on the Fediverse and LinkedIn.
+You can follow our official social media accounts which we monitor.
```
I'd drop 'which we monitor'.  I think it's clear, especially since below we mention we don't monitor the platform formerly known as Twitter.

stumbo commented on content/en/project/getInvolved/_index.md:
```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
 
+You can email us at [info@interlisp.org](mailto:info@interlisp.org)
+
 * Interlisp.org is a Google workspace; [let us know](mailto:info@interlisp.org) if you want an "interlisp.org" email address and why you want it.
```
I realize this isn't a change you made, but...

Do we really want to advertise **interlisp.org** email addresses?  Shouldn't that be something we give people who are active within the group?

stumbo commented on content/en/project/getInvolved/_index.md:
```diff
@@ -28,40 +28,44 @@ Bug reports, feature requests, documentation problems, and questions are posted
-### Chat
+### Online communities
+
+We do not monitor these online communities and we are looking for volunteers who can monitor them for Interlisp discussions and keep us in the loop:
```
How about a little massaging of the text:
>We do not monitor these online communities but are looking for volunteers to monitor them for Interlisp discussions....

pamoroso committed:
Minor revisions of page Get Involved

Revise the page Get Involved. Under GitHub, capitalize Maiko and make the interlisp.org entry consistent with the others, and link to the Medley Online repo. Drop the reference to monitored accounts from section Social media accounts. Remove reference to Google Workspace accounts from section EMail. Reword the opening text of section Online communities for better flow.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>
